core_gem/lib/deep_cover/autoload_tracker.rb

```diff
@@ -52,7 +52,7 @@ def wrap_require(requested_path, absolute_path_found) # &block
         entries.each do |entry|
           mod = entry.mod_if_available
           next unless mod
-          mod.autoload_without_deep_cover(entry.name, already_loaded_feature)
+          mod.send("autoload_without_deep_cover_#{ModuleOverride.object_id}", entry.name, already_loaded_feature)
```

Comment:
👎  IMO. Why can't we use the instrumented code?

Reply:
I find it a nice goal that we could use any version of DeepCover to cover any other version of DeepCover. If we manage that, then I have more confidence in the general quality of the system.

That being said, I had an idea of how to avoid having to do that in every case. (And only needing to do that in the `exec` extension).